### PR TITLE
vimPlugins.zk-nvim: init at 2022-3-15

### PR DESCRIPTION
**Pull request by notusknot**
Added zk-nvim plugin to vimPlugins

### CODE REVIEW

File: pkgs/applications/editors/vim/plugins/vim-plugin-names [hunk 1]
```diff
@@ -501,6 +501,8 @@ mhinz/vim-sayonara@7e774f58c5865d9c10d40396850b35ab95af17c5
 mhinz/vim-signify
 mhinz/vim-startify
 michaeljsmith/vim-indent-object
+mickael-menu/zk-nvim
+mickael-menu/zk-nvim
```
Comment (SuperSandro2000) [inline pkgs/applications/editors/vim/plugins/vim-plugin-names:505]:
```suggestion
```

File: pkgs/applications/editors/vim/plugins/generated.nix [hunk 2]
```diff
@@ -12026,16 +12026,40 @@ final: prev:
...
+    pname = "zk-nvim";
+    version = "2022-02-13";
+    src = fetchFromGitHub {
+      owner = "mickael-menu";
+      repo = "zk-nvim";
+      rev = "d705faa82da042e7a75f7d244afd27c2d8f20830";
+      sha256 = "0kgk9aa85bry9f6njff53z4gy62h2m57bcjwarkls8rd6gq92szz";
+    };
+    meta.homepage = "https://github.com/mickael-menu/zk-nvim/";
+  };
```
Comment (SuperSandro2000) [inline pkgs/applications/editors/vim/plugins/generated.nix:12061]:
```suggestion
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/applications/editors/vim/plugins/generated.nix (hunk 2 above)
Comment (notusknot) [reply pkgs/applications/editors/vim/plugins/generated.nix:12061]: Fixed

**Comment by notusknot** [comment]
Fixed

**Comment by SuperSandro2000** [comment]
Please rebase to solve the merge conflict.

**Comment by notusknot** [comment]
I redid it and it should be good now

**Pull request merged**